library_management_system_downloader/downloader_configuration_file_TEMPLATE.py

```diff
@@ -0,0 +1,18 @@
+# These are parameters that stay the same from query to query (unlike the
```

Comment:
This is a nice template. The non-template version should be ignored (added to `.gititnore`) right? To prevent accidental tracking.

Also you should add to the `.gitignore` any other files that are generated by are not intended to be tracked with git.

Reply:
I've added `.gitignore` in 2e939. I'd had `.gitignore` listed in the file, and hadn't been tracking it until now.